### PR TITLE
test: fix randomly failing E2E tests

### PR DESCRIPTION
**Pull request by arnautov-anton**
### 🎯 Goal

Get rid of excessive `setState` action in `ConnectedUser` component and adjust Playwright configuration.

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ import type { PlaywrightTestConfig } from '@playwright/test';
 // https://playwright.dev/docs/test-configuration
 const config: PlaywrightTestConfig = {
   testDir: './e2e',
+  timeout: 15 * 1000,
   use: {
-    headless: true,
```
Comment (petyosi) [inline playwright.config.ts:7]: Should not this be explicitly set? Or it's the default value.

Comment (MartinCupela) [reply playwright.config.ts:7]: The default value is `false`. If removed, the browser windows do not appear.

**Review by MartinCupela — APPROVED** [review]

**Pull request merged**